src/storage/exec/QueryUtils.h

```diff
@@ -86,6 +86,22 @@ class QueryUtils final {
         return Status::Error(folly::stringPrintf("Invalid property %s", prop.name_.c_str()));
     }
 
+    static Status collectPropsInValue(RowReader* reader,
+                                      const std::vector<PropContext>* props,
+                                      nebula::List& list) {
+        for (const auto& prop : *props) {
+            if (prop.returned_ && prop.propInKeyType_ == PropContext::PropInKeyType::NONE) {
+                VLOG(2) << "Collect prop " << prop.name_;
+                auto value = QueryUtils::readValue(reader, prop.name_, prop.field_);
```

Comment:
Do we need to determine the default values?

Reply:
Yeah, default values has been handled in `readValue`

Reply:
> Yeah, default values has been handled in `readValue`

Got, excellent.